app/controllers/works_controller.rb

```diff
@@ -102,8 +102,12 @@ def index
       end
 
       tag = @fandom || @tag
-      options[:filter_ids] ||= []
-      options[:filter_ids] << tag.id
+      # This strange dance is because there is an interaction between
+      # strong_parameters and dup, with out the dance 
```

Comment:
I don't know if it's a temporary hack or a permanent change. I guess it is dependant on whether the strong params gem and rails 4 implement it in the same way. I would add it to the list of things to test after the rails 4 upgrade.